### PR TITLE
📝  Textarea documentation

### LINKED ISSUE #2291: Create documentation for Textarea

### PR DESCRIPTION
**Pull request by martalalik**
Resolves #2291

### CODE REVIEW

File: packages/eds-core-react/src/components/Textarea/Textarea.docs.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Links, PropsTable, Story } from './../../../.storybook/components'
+import { Textarea } from '.'
+
+# Textarea
+
+Helper component - is used for collecting large amounts of textual data.
```
Comment (oddvernes) [inline packages/eds-core-react/src/components/Textarea/Textarea.docs.mdx:6]: Should maybe make it more clear this is mostly internal and most user should just use TextField

"Helper component developed mainly for internal use. Can be used as a building block for users requiring more flexibility than TextField provides" 
Something like this?

Comment (martalalik) [reply packages/eds-core-react/src/components/Textarea/Textarea.docs.mdx]: Updated

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Review by oddvernes — APPROVED** [review]
LGTM

**Pull request merged**